### ISSUE TITLE
Revision mismatch, unclear what I should do

### ISSUE THREAD

**Issue by dpiddy** [tools/godep#162]
Getting this:

```
% godep save
godep: github.com/stripe/aws-go/gen/dynamodb: revision is 89ffcf19352c19cb983f97e293bed2c05ad805ab, want c16321324a5e5c34ca14ab869ec99cab10ee62cc
```

what should I do about it? `github.com/stripe/aws-go/gen/dynamodb` is not being added to `Godeps/Godeps.json`. I have other things from github.com/stripe/aws-go and they are all at `c16321324a5e5c34ca14ab869ec99cab10ee62cc`, so not sure where `89ffcf19352c19cb983f97e293bed2c05ad805ab` is coming from.

**Comment by brandur** [comment]
I ran into this as well trying to update dependencies and was able to resolve it in the same way as Dan. I'm not 100% clear why Godep wants a particular (old) revision here when it's trying to run an update to whatever the latest is.

**Comment by freeformz** [comment]
@brandur what project / revision was this on ?

**Comment by brandur** [comment]
Hey @freeformz, thanks for following up!

The project was [heroku/holm](https://github.com/heroku/holm), but I've since managed to resolve it.

I played around with Godep more this weekend and I believe that this is related to #138 in that it's caused by pulling in multiple packages that come from the same repo (which fits with Dan's problem as well because this is how aws-go works). To correct what I said previously about Dan's suggestd fix, as time went on I found that this didn't always work for me, what does work with good regularity is run to `godep update` with the spread operator on the top-level repository as suggested in the other thread:

```
godep update github.com/stripe/aws-go/...
```
